app/src/screens/dev/MockDataScreen.tsx

```
@@ -38,24 +44,120 @@ import {
+  slate500,
 } from '../../utils/colors';
 import { extraYPadding } from '../../utils/constants';
 import { buttonTap, selectionChange } from '../../utils/haptic';
 
+import { Caption } from '../../components/typography/Caption';
+import { dinot, plexMono } from '../../utils/fonts';
+
+
 const { trackEvent } = analytics();
 
 interface MockDataScreenProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)